crypto/ctype.c

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright 2017 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <string.h>
+#include <e_os.h>
```

Comment:
Already discussed. What's in <e_os.h>? Is this module *that* OS-specific? [For example, does it have to #include <winsock.h> and <windows.h>?]